src/components/Board.js

```diff
@@ -31,9 +31,13 @@ const Board = ({ socket }) => {
 
   socket.on('move_done', (data) => {
     console.log("From Server" + JSON.stringify(data));
+    if(data.status === 'success'){
+      setBoard(data.boardState);
```

Comment:
i think cera is doing this in her pr, so maybe remove this

Reply:
Yes, I already have the  board states correctly showing in the front-end. I think we should leave it the way it is. Otherwise looks good.